.github/workflows/unwedge-maven-central.yml

```diff
@@ -0,0 +1,56 @@
+# Although our CI builds should automatically close and release repositories after publishing, sometimes
+# this process can fail.  This GitHub Action allows a team member to manually unwedge this stuck deployment.
+
+# Expected secrets
+# MAVEN_CENTRAL_USERNAME - Username for Maven Central
+# MAVEN_CENTRAL_PASSWORD - Password for Maven Central
+
+name: Deploy Snapshot
+
+on:
+  workflow_dispatch:
+    inputs:
+      mavenCentralRepository:
+        description: 'Repository name to close'
+        required: true
+
+concurrency: deploy_snapshot
+
+jobs:
+  validate_gradle_wrapper:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        timeout-minutes: 1
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
```

Comment:
Will we need to update this commit hash in the future, e.g. when new versions of actions/checkout come out? If yes, please make sure that's tracked in a ticket.

Reply:
This would be true for all actions that we consume (we migrated to SHAs last week).  My plan was to open a ticket whenever a new version is released.

Did you have something else in mind?